spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -128,6 +134,10 @@ Then("{string} label should be unchecked", (text) => {
     });
 });
 
+Then("the table should have {int} row(s)", (number) => {
+  cy.get("tbody").find("tr").should("have.length", number);
+});
```

Comment:
Is there a way of making this more generic? E.g. `there should be {int} instances of {string}`. wording feels clunky though

Reply:
I'd like to leave this for now and commonise if and when we find a need. I think if I try to do it now, it won't be quite right when we come to use it

Reply:
Also, here we specifically need tbody > tr, since there is a tr in the header

Reply:
Although I can do that with css selectors...

Reply:
My problem with that is that it assumes the next person who needs to do something similar is aware of this conversation!

I'm not massively fussed either way :)

Reply:
I shall endeavour to make them aware through review 😛